src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -499,5 +499,11 @@ public static unsafe float Int32BitsToSingle(int value)
 
             return *((float*)&value);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe short HalfToInt16Bits(Half value)
```

Comment:
internal because we use it from Half.cs

Reply:
We should get an API proposal up for this as well.

Reply:
(and for the counterpart `Int16BitsToHalf`)

Reply:
https://github.com/dotnet/runtime/issues/38288